matrix_sdk/src/verification/mod.rs

```diff
@@ -30,11 +30,13 @@
 //! string.
 //! * [`QrVerification`] - Interactive verification using QR codes.
 
+#[cfg(feature = "qrcode")]
 mod qrcode;
 mod requests;
 mod sas;
 
 pub use matrix_sdk_base::crypto::{AcceptSettings, CancelInfo};
+#[cfg(feature = "qrcode")]
```

Comment:
You'll have to a a doc mark  that this is only available using this feature like we have for other features, e.g. for the encryption feature:

```rust
/// Get the public ed25519 key of our own device. This is usually what is
/// called the fingerprint of the device.
#[cfg(feature = "encryption")]
#[cfg_attr(feature = "docs", doc(cfg(encryption)))]
pub async fn ed25519_key(&self) -> Option<String> {
    self.base_client.olm_machine().await.map(|o| o.identity_keys().ed25519().to_owned())
}
```

This needs to be added to all the public structs and methods that get disabled by this feature flag.